text/0000-futures.md

```diff
@@ -105,13 +105,21 @@ In addition to `async fn` definitions, futures can be built using adapters, much
 like with `Iterator`s. Initially these adapters will be provided entirely "out
 of tree", but eventually they will make their way into the standard library.
 
-Ultimately asynchronous computations are executed by *tasks*, which are
-lightweight threads. In particular, an *executor* is able to "spawn" a
-`()`-producing `Future` as an independent task; these tasks are then
-cooperatively scheduled onto one or more operating system threads. The
-`Executor` trait defines this interface, and the `task` module provides a host
-of related definitions needed when manually implementing `Future`s or
-executors.
+Ultimately asynchronous computations are executed in the form of *tasks*,
+which are comparable to lightweight threads. A task is a `()`-producing `Future`,
```

Comment:
> A task is a `()`-producing `Future`

This was a [point of contention](https://github.com/rust-lang-nursery/futures-rs/issues/1087) back when 0.3 was being implemented, the [final verdict](https://github.com/rust-lang/rust/pull/52610) was that _the future is not the task_, the task is a concept on top of the future that is executor specific and not directly exposed in any APIs (indirectly the APIs under `core::task` are used by the executor to communicate with the future that is at the root of the task).

Reply:
Yes, it's true. The idea was to describe that a task is some kind of running version of a `Future` that is owned by the executor and accompanied by some extra state. Will try to make it clearer.

Reply:
I updated the wording, and think it's better now. But I would also be fine with the one that MajorBreakfast added in 
https://github.com/rust-lang/rust/pull/52610